frontend/src/Signin.js

```diff
@@ -0,0 +1,49 @@
+import styles from './auth.module.css';
+import classNames from 'classnames';
+//<img src={logo} className="App-logo" alt="logo" /> old logo
+//<header className="App-header">
+//        Hello
+//      </header>
+// 
+function Signin() {
+  return (
+    <div className = {styles.auth}>
+      <div className = {styles.main}>
+        <div className = {styles.border}>
+        <header className = {styles.app}>
+                <a href="./"><img src="career_connect_logo.png" alt="logo" width ="200" hieght="200" /></a>
+                <br />
+                    Welcome back!
+                <br />
+            </header>
+            <br />
+            <header className = {styles.describeAuth}>Enter your login details below</header>
+            <br/>
```

Comment:
To improve the semantics of this SignIn form, consider encapsulating your input fields in a `<form>` element here. It will also make easier to handle form submissions later on.